### DISCUSSION TITLE
Multiple selection for mass moving of Issues/discussion

### DISCUSSION THREAD

**Discussion by TomLewis** [community/community#2963] [Discussions]
Ahoy

With the introduction of discussions, some of us now require to move thousands and thousands of Issues into Questions/Ideas discussion boards. Can we please get tools to mass select and copy/delete Issues/discussions at once, so we don't have to go through manually every single issue and move it...

**Comment by becca** [comment]
Thank you so much for this feedback. We are now tracking this feature request and will do our best to keep you posted on when this feature lands. Also, be sure to keep you eye on our [public roadmap](github.com/github/roadmap) for more updates!
